test/models/api2_test.rb

```diff
@@ -1747,7 +1777,9 @@ def test_getting_names
     assert_api_pass(params.merge(is_deprecated: "true"))
     assert_api_results(names)
 
-    names = Name.where("date(updated_at) = '2009-10-12'")
+    # rubocop:disable Style/FormatStringToken
+    names = Name.where(Name[:updated_at].format("%Y-%m-%d") == "2009-10-12")
```

Comment:
It seems to me that there should be an AR way to do a simple equals condition.

Name.where(updated_at: Date.parse("2009-10-12"))

or something like that?

Reply:
I'll try it.

Reply:
That doesn't work, unfortunately. It generates this, which does not pass the parse to the SQL.
```
"SELECT `names`.* FROM `names` WHERE `names`.`updated_at` = '2009-10-12'"
```
The way i've got it was the recommended short syntax from Arel extensions. It generates:
```
"SELECT `names`.* FROM `names` WHERE DATE_FORMAT(`names`.`updated_at`, '%Y-%m-%d') = '2009-10-12'"
```
The extra format seems extraneous, but more foolproof in the event of any future db change.

Reply:
Oh, right, thanks for the explanation.  I think what I was missing is that updated_at is date-time, so this is not actually an "equals condition" like I'd assumed.